### PR TITLE
add notes for PKS k8s cluster installations

### PR DESCRIPTION
**Pull request by saravanan30erd**
- add notes for PKS k8s cluster installations.

### CODE REVIEW

**Comment by cwen0** [comment]
@saravanan30erd Thanks for your contribution! This note is very useful for users!

**Review by cwen0 — APPROVED** [review]
LGTM

File: docs/user_guides/installation.md [hunk 1]
```diff
@@ -106,6 +106,15 @@ Depending on your environment, there are two methods of installing Chaos Mesh:
      helm install chaos-mesh chaos-mesh/chaos-mesh --namespace=chaos-testing
      ```
 
+     > **Note:**
+     >
+     > Sometimes we might need to change the default values based on Kubernetes editions.
+     > Example, Docker socket path is different in PKS Kubernetes clusters (VMware K8s Enterprise editions).
```
Comment (dcalvin) [inline docs/user_guides/installation.md:112]:
```suggestion
     >Depending on your Kubernetes edition, you may have to change the default values.
     > For example, the Docker socket path is different in PKS Kubernetes clusters (VMware K8s Enterprise edition).
```

Comment (saravanan30erd) [reply docs/user_guides/installation.md]: @dcalvin  changes are done based on feedback.

**Review by dcalvin — APPROVED** [review]
LGTM. @saravanan30erd Thanks for your contribution

**Pull request merged**